AcadeMeData/test_messages.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from AcadeMeData.models import Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.fixture
+def generate_message(user_example, msgID=1, text='This is a test message yo'):
```

Comment:
this fixture can call user_example fixture
please remove line 16

Reply:
Again its as matter of style, nothing here isnt correct.
putting the fixture user_example in a variable user is the way I chose to do it
Even the space complexity (which is the only thing that comes to mind) is negligible!